usr/src/cmd/fs.d/nfs/clear_locks/clear_locks.c

```diff
@@ -24,6 +24,10 @@
  * Use is subject to license terms.
  */
 
+/*
+ * Copyright 2023 MNX Cloud, Inc.
```

Comment:
Maybe this is pedantic, but shouldn't this be right after line 23?

Reply:
There are two schools of thought on this.  One is same block, the other is new block.  I'm going with the latter mostly to avoid the "Use is subject to license terms" that we would've been told to eliminate had we touched something post-2007-ish after jumping to CDDL 1.0.

Reply:
I can get on board with this line of thought.